src/typescript/index.js

```diff
@@ -0,0 +1,102 @@
+import got from 'got';
+
+import config from '../config.js';
+import datadog from '../datadog.js';
+import log from '../log.js';
+
+const typesCache = {};
+
+/**
+ * Microsoft build a index.json with all @types/* on each publication
+ * https://github.com/microsoft/types-publisher/blob/master/src/create-search-index.ts
+ *
+ */
+async function loadTypesIndex() {
+  const start = Date.now();
+  const { body } = await got(config.typescriptTypesIndex, {
+    decompress: true,
+    json: true,
+  });
+
+  log.info(`📦  Typescript preload, found ${body.length} @types`);
+
+  // m = modules associate
+  // t = @types/<name>
+  body.forEach(type => {
+    type.m.forEach(m => {
+      typesCache[m] = type.t;
```

Comment:
It will work (if you look [here](https://typespublisher.blob.core.windows.net/typespublisher/data/search-index-min.json) it's already correctly name)
but you example made me found out that not all packages as a property `m` in the json

Reply:
For example `mapbox__polyline` does not contain the correct package name, `reach__router` seems to have the same issue. I was expecting to have the correct package name in `m` but it seems not consistant.
(maybe @sandersn has an idea) 
If no solution I will unescape the name probably to transform the `__` into a `/`

Reply:
I'm pretty sure that the module name is supposed to be the DT-mangled name, eg `mapbox__polyline` instead of `@mapbox/polyline`. Do you see unmangled package names too?

Reply:
For example: `@mapbox/geojson-area` or `arangodb` appear as-is in the `m` property.
I'm really not sure what should or should not be mangled